index.html

```diff
@@ -124,8 +125,8 @@ <h2>Watching a video in a loud room, reading in Braille, writing using an altern
                             <div class="floe-image floe-image-UIO"></div>
                             <h2>Want to select, refine and apply your preferences?</h2>
                             <p>Preference editing tools help learners personalize their learning experience.
-                               <a href="http://build.fluidproject.org/infusion/demos/prefsFramework/">Learner Options</a>
-                               allow selection and fine-tuning of preferences.</p>
+                               <a href="http://build.fluidproject.org/infusion/demos/prefsFramework/">UI Options</a>
+                               allow selection and fine-tuning of preferences. <a href="./ui-options.html">Learn more</a> about adding it to your site or application.</p>
```

Comment:
I might be off with the grammar hear, but should it be "allows" instead of "allow" because UI Options is the name of a singular thing. I think this link describes what I mean better, http://www.learnersdictionary.com/qa/Inflections-When-to-Add-S-to-Regular-Verbs
```suggestion
                               allows selection and fine-tuning of preferences. <a href="./ui-options.html">Learn more</a> about adding it to your site or application.</p>
```